sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1119,4 +1123,16 @@ private String getLinkName(String sessionId) {
             return existing != null ? existing.getLinkName() : null;
         }
     }
+
+    //TODO: discuss whether we should disallow session id call on peek, receiveDeferred.
```

Comment:
Should we?

Reply:
latest update: removed these sessionId-related APIs because `ServiceBusReceiverClient` doesn't support multi-session any more.